src/main/java/wolox/training/controllers/UserController.java

```diff
@@ -68,34 +77,33 @@ public User create(@RequestBody User user) {
     /**
      * Delete an existing user.
      *
-     * @throws UserNotFoundException
+     * @throws EntityNotFoundException
      * @param id the path variable id of the user to find and delete
      */
     @DeleteMapping("/{id}")
     @ResponseStatus(HttpStatus.NO_CONTENT)
     public void delete(@PathVariable Long id) {
         userRepository.findById(id)
-            .orElseThrow(UserNotFoundException::new);
+            .orElseThrow(() -> new EntityNotFoundException(User.class));
         userRepository.deleteById(id);
     }
 
     /**
      * Updates an existing user.
      *
      * @throws IdMismatchException if pathVariable id does not match body id.
-     * @throws UserNotFoundException
+     * @throws EntityNotFoundException
      * @param user the request updated user to save
      * @param id   the path variable for the user id
      * @return the saved updated user
      */
     @PutMapping("/{id}")
     @ResponseStatus(HttpStatus.NO_CONTENT)
     public void updateUser(@RequestBody User user, @PathVariable Long id) {
-        if (user.getId() != id) {
+        if (user.getId() != id)
```

Comment:
Just a comment here: It's better to use the brackets on one line ifs
[Link](https://docs.google.com/document/d/1xBoy41sG9vkrVsOd109OQ61LBQxPzZ2_F3K9e5jfHQY/edit?ts=5e4db74c#heading=h.4bnl8u3wizvm)